.clippy_allowed_lints

```diff
@@ -0,0 +1,39 @@
+# This is a list of allowed Clippy rules for the purposes of gradual migration.
```

Comment:
Out of curiosity, how many existing violations did this find? 😅

Reply:
I didn't count :smile: The vast majority is `needless_return`, most of them are a single-digit occurences, so the lint fix diff is not so bad!